wg-data/charter.md

```diff
@@ -0,0 +1,66 @@
+# WG Data Charter
+
+This charter adheres to the conventions, roles, and organisation management outlined in [wg-governance] for the Working Group "Data".
+
+## Scope
+
+The WG "Data" is focused on enhancing the support for Data/metadata-related tasks within Kubeflow, with a specific focus on the [Spark operator](https://github.com/kubeflow/community/pull/672) and [Model Registry](https://github.com/kubeflow/kubeflow/issues/7396).
+The group aims to streamline data processing workflows, facilitate efficient data lifecycle and ML models' metadata management, while ensuring seamless integration with other Kubeflow components.
+
+An additional goal of the group is to offer a common ground for data/metadata-related topics in the MLOps orbit that didn't have a more specific working group yet, so they can "incubate as one", coherent effort.
+
+For example: Data Preparation, Feature Store, and Model Registry have been recently discussed in the Kubeflow community while not mature enough yet to have their own working group, they can be nurtured together as part of this WG.
+
+### In scope
+
+#### Code, Binaries, and Services
+
+- Onboarding and maintenance of the Spark operator for scalable and distributed data processing.
+[See also](https://github.com/kubeflow/community/pull/672)
+- Continued development of the Model Registry to manage and version machine learning models efficiently.
+[See also](https://github.com/kubeflow/kubeflow/issues/7396)
+- CLI tools and REST APIs for interacting with Kubeflow APIs related to data processing and ML models metadata management.
```

Comment:
When you mention CLI and REST APIs what do mean exactly ?

Reply:
reworded "CLI" as SDK with 61ddf2396ab928bd6db199f1cd3590aee15b08d4

for instance for Model Registry, we have both a [REST API](https://editor.swagger.io/?url=https://raw.githubusercontent.com/kubeflow/model-registry/main/api/openapi/model-registry.yaml) and a [Python client](https://github.com/kubeflow/model-registry/tree/main/clients/python#readme) (so reworded as SDK here).

Reply:
I know that Spark offers `sparkctl` CLI to interact with SparkJobs.
@vara-bonthu @yuchaoran2011 @ChenYi015 Should we mention it in the binaries that WG is going to maintain ?